.github/workflows/build-test.yaml

```diff
@@ -1788,8 +1788,7 @@ jobs:
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
 
 
-  # this job will validate that all validate-* jobs succeed and is used for the github branch protection rule
-  validate-success:
+  validate-pr-tests:
```

Comment:
We have a bug and I think it is caused by change the name of the test/check.
Could we keep the same name?

Reply:
I believe `validate-success` fails because the `validate-kurl-addon` step was skipped and caused `validate-pr-tests` to be skipped as well.  I've pushed another commit to change this so that `validate-kurl-addon` is a checked within `validate-success` since it may or may not run depending on if changes were made to the add-on.